canonical/cloud-init
# includedir in suoders can be prefixed by "arroba"

## Proposed Commit Message


```
Since version 1.9.1, @includedir can be used in the sudoers files
instead of #includedir:

https://github.com/sudo-project/sudo/releases/tag/SUDO_1_9_1

This happens to be the default in SUSE Linux enterprise sudoer package,
so cloudinit should take this into account.

Otherwise, cloudinit was adding an extra #includedir, which was
resulting on the files under /etc/sudoers.d being included twice, one by
@includedir from the SUSE package, one by the @includedir from
cloudinit. The consequence of this, was that if you were defining an
Cmnd_Alias inside any of those files, this was being defined twice and
creating an error when using sudo.

Signed-off-by: Jordi Massaguer Pla <jmassaguerpla@suse.de>```

## Additional Context


## Test Steps


Add "@includedir /etc/sudoers" into /etc/sudoers

Apply cloud-init and see you have both, "@includedir /etc/sudoers" and "#includedir /etc/sudoers".

After the fix, you should only have "@includedir"

## Checklist:
